src/components/resourceLinks/ResourceLinks.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import "./ResourceLinks.css"
+const ResourceLinks = ({icon, icon_alt, target, url, icon_only, name, classname}) => {
```

Comment:
De properties geven we altijd door in camelCase! En als dingen _booleans_ zijn, is het handig om daar een vraag van te maken. Dus in plaats van `icon_only` maak je er de property `isOnlyIcon` van, want dat leest gemakkelijker!

Reply:
Je kunt ook denken: als iemand een icon meegeeft, dan betekent dat waarschijnlijk een `isIconOnly` dus dan hoef je eigenlijk alleen maar te checken op het icoon, en als ie er is, geef je 'm weer